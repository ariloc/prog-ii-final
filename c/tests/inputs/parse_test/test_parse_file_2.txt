Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus cursus diam sit amet euismod hendrerit. In malesuada diam ante. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Integer mattis ac libero sollicitudin varius. Suspendisse aliquam et libero vitae rutrum. In in blandit nulla. Nam mattis leo at magna facilisis, non tempus tortor elementum. Lorem ipsum dolor sit amet, consectetur adipiscing elit.

Duis molestie arcu ut dapibus lacinia. Fusce ac lectus ac lorem egestas sollicitudin. Pellentesque dapibus libero at ex fringilla varius. Curabitur sit amet tincidunt libero. Duis aliquet felis vitae dapibus feugiat. Aenean suscipit, est feugiat imperdiet hendrerit, ante est tincidunt purus, et iaculis ipsum enim ut lorem. Donec convallis felis nec orci lacinia vestibulum. Nulla pulvinar, massa eget ultricies mattis, turpis eros eleifend ligula, ut aliquet erat metus vel nisl. Proin in lacinia nisi, ac rutrum leo. Fusce pellentesque enim at elit bibendum, nec lacinia ligula pretium. Aenean rhoncus porttitor tortor vitae pharetra. In consequat tincidunt neque tempus aliquam. Donec elementum dui ac dui rhoncus tristique.

Curabitur pharetra velit a placerat congue. Vivamus mollis mauris ac risus volutpat iaculis. Suspendisse euismod elit neque, sed bibendum arcu suscipit in. Phasellus tortor purus, hendrerit eu turpis eu, convallis volutpat enim. Mauris cursus odio eu massa porta, ut congue ligula commodo. Duis at porta mauris, vitae mattis magna. Ut ultrices, nisi ac facilisis iaculis, metus massa dapibus sem, ac dapibus dui leo et mauris. Mauris porttitor turpis eros, a iaculis sem porta sed.

Aliquam eget purus porta, tempus orci et, pulvinar lorem. Proin placerat ante nec elementum iaculis. Nulla facilisi. Integer eget pellentesque risus. Vivamus gravida risus in pulvinar feugiat. Nullam ut ligula eget lorem malesuada convallis sit amet non erat. Vivamus et laoreet libero. Vivamus viverra gravida arcu quis faucibus.

Donec a velit eu quam rutrum accumsan. Nulla nec tempor est. Praesent a massa vel justo cursus sodales vel quis erat. Praesent ornare orci sed tristique auctor. Nulla at fringilla massa. Pellentesque luctus malesuada orci, tincidunt scelerisque nulla luctus pharetra. Nunc laoreet est id justo bibendum, quis semper magna malesuada. Praesent sed lorem vulputate, volutpat tellus et, imperdiet leo. Vestibulum faucibus, ligula at venenatis consequat, tellus quam fringilla leo, dignissim accumsan augue nunc vel purus. Aenean et lorem ut leo consectetur hendrerit a id nunc. Sed semper ipsum dolor, vel vestibulum leo venenatis vel. Duis consectetur, mi ac eleifend porta, nibh metus convallis sapien, sed hendrerit dui felis et enim. 