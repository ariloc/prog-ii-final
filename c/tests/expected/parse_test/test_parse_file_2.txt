lorem ipsum dolor sit amet consectetur adipiscing elit
vivamus cursus diam sit amet euismod hendrerit
in malesuada diam ante
vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae integer mattis ac libero sollicitudin varius
suspendisse aliquam et libero vitae rutrum
in in blandit nulla
nam mattis leo at magna facilisis non tempus tortor elementum
lorem ipsum dolor sit amet consectetur adipiscing elit
duis molestie arcu ut dapibus lacinia
fusce ac lectus ac lorem egestas sollicitudin
pellentesque dapibus libero at ex fringilla varius
curabitur sit amet tincidunt libero
duis aliquet felis vitae dapibus feugiat
aenean suscipit est feugiat imperdiet hendrerit ante est tincidunt purus et iaculis ipsum enim ut lorem
donec convallis felis nec orci lacinia vestibulum
nulla pulvinar massa eget ultricies mattis turpis eros eleifend ligula ut aliquet erat metus vel nisl
proin in lacinia nisi ac rutrum leo
fusce pellentesque enim at elit bibendum nec lacinia ligula pretium
aenean rhoncus porttitor tortor vitae pharetra
in consequat tincidunt neque tempus aliquam
donec elementum dui ac dui rhoncus tristique
curabitur pharetra velit a placerat congue
vivamus mollis mauris ac risus volutpat iaculis
suspendisse euismod elit neque sed bibendum arcu suscipit in
phasellus tortor purus hendrerit eu turpis eu convallis volutpat enim
mauris cursus odio eu massa porta ut congue ligula commodo
duis at porta mauris vitae mattis magna
ut ultrices nisi ac facilisis iaculis metus massa dapibus sem ac dapibus dui leo et mauris
mauris porttitor turpis eros a iaculis sem porta sed
aliquam eget purus porta tempus orci et pulvinar lorem
proin placerat ante nec elementum iaculis
nulla facilisi
integer eget pellentesque risus
vivamus gravida risus in pulvinar feugiat
nullam ut ligula eget lorem malesuada convallis sit amet non erat
vivamus et laoreet libero
vivamus viverra gravida arcu quis faucibus
donec a velit eu quam rutrum accumsan
nulla nec tempor est
praesent a massa vel justo cursus sodales vel quis erat
praesent ornare orci sed tristique auctor
nulla at fringilla massa
pellentesque luctus malesuada orci tincidunt scelerisque nulla luctus pharetra
nunc laoreet est id justo bibendum quis semper magna malesuada
praesent sed lorem vulputate volutpat tellus et imperdiet leo
vestibulum faucibus ligula at venenatis consequat tellus quam fringilla leo dignissim accumsan augue nunc vel purus
aenean et lorem ut leo consectetur hendrerit a id nunc
sed semper ipsum dolor vel vestibulum leo venenatis vel
duis consectetur mi ac eleifend porta nibh metus convallis sapien sed hendrerit dui felis et enim
